netbeans/libxtp/nbproject/private/private.xml

```diff
@@ -4,55 +4,83 @@
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/jobcalculators/idft.cc</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/calculators/kmcmultiple.h</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/aomatrices/aomatrix.cc</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/diis.cc</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/threecenters_tools.cc</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/calculators/kmcmultiple.cc</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/gwbse/gwbse.cc</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/include/votca/xtp/orbitals.h</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/overlap.cc</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/calculators/kmclifetime.h</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/orbitals.cc</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/qmpackages/nwchem.cc</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/qmpackage.cc</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/include/votca/xtp/qmapemachine.h</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/bsecoupling.cc</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/gridbox.cc</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/include/votca/xtp/kmccalculator.h</file>
-            <file>file:/home/jens/Programs/votca/src/xtp/src/libxtp/calculators/kmclifetime.cc</file>
+            <file>file:/home/yuriy/soft/votca/src/xtp/include/votca/xtp/bulkesp.h</file>
+            <file>file:/home/yuriy/soft/votca/src/xtp/src/libxtp/bulkesp.cc</file>
+            <file>file:/home/yuriy/soft/votca/src/xtp/src/libxtp/espfit.cc</file>
+            <file>file:/home/yuriy/soft/votca/src/xtp/include/votca/xtp/aobasis.h</file>
+            <file>file:/home/yuriy/soft/votca/src/xtp/include/votca/xtp/grid.h</file>
+            <file>file:/home/yuriy/soft/votca/src/xtp/src/libxtp/numerical_integration/numerical_integrations_periodic.cc</file>
+            <file>file:/home/yuriy/soft/votca/src/xtp/src/libxtp/numerical_integration/numerical_integrations.cc</file>
+            <file>file:/home/yuriy/soft/votca/src/xtp/include/votca/xtp/numerical_integrations_periodic.h</file>
+            <file>file:/home/yuriy/soft/votca/src/xtp/include/votca/xtp/numerical_integrations.h</file>
+            <file>file:/home/yuriy/soft/votca/src/xtp/src/libxtp/grid.cc</file>
+            <file>file:/home/yuriy/soft/votca/src/xtp/include/votca/xtp/gridbox.h</file>
+            <file>file:/home/yuriy/soft/votca/src/xtp/include/votca/xtp/espfit.h</file>
+            <file>file:/home/yuriy/soft/votca/src/xtp/src/libxtp/gridbox.cc</file>
```

Comment:
Huh, `home/jens` -> `home/yuriy`? I think we should really move these netbeans files out of version control.